packages/IMAPClient-UI.package/ICLoginDialog.class/instance/login.st

```diff
@@ -16,6 +16,7 @@ login
 		do: [ :sig | self inform: sig asString ].
 	(endpoint isLoggedIn)
 		ifTrue: [
+			self loginMorph abandon.
 			ICFolderDialog openWith: endpoint. 
 			Transcript show: (self class successLoginMessage); cr ] 
 		ifFalse: [ self inform: self class failureLoginMessage ].
```

Comment:
You see, here you leave all the spaces the way the team before us did it. I think we should stick to that style.